examples/misc/upload_media_bundle.py

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env python
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""This example uploads an HTML5 zip file as a media bundle."""
+
+
+import argparse
+import requests
+import sys
+
+
+from google.ads.google_ads.client import GoogleAdsClient
+from google.ads.google_ads.errors import GoogleAdsException
+
+
+BUNDLE_URL = 'https://goo.gl/9Y7qI2'
+
+def main(client, customer_id):
+    media_file_operation = client.get_type('MediaFileOperation', version='v2')
+    media_file = media_file_operation.create
+    media_file.name.value = 'Ad Media Bundle'
+    media_file.type = (client.get_type('MediaTypeEnum',
+                                       version='v2').MEDIA_BUNDLE)
+    # Download the ZIP as bytes from the URL
+    media_file.media_bundle.data.value = requests.get(BUNDLE_URL).content
+
+    media_file_service = client.get_service('MediaFileService', version='v2')
+
+    try:
+        mutate_media_files_response = (
```

Comment:
Would add a comment

Reply:
We are trying to avoid comments that restate the obvious (see Mark's prior comment about the docstring).

Reply:
SG